lib/bowling.rb

```diff
@@ -0,0 +1,128 @@
+class Bowling
```

Comment:
I think I'd consider the responsibilities of this class: At the moment, it's responsible for co-ordinating the game (calling methods at the right time etc), interacting with the user to obtain rolls and outputting the score, creating the score sheet and determining what should happen in each frame based on user input. As you said, you could definitely extract this into one more class, but it might be worth thinking about two further classes.